Framework/Core/include/Framework/Configurable.h

```diff
@@ -95,6 +96,9 @@ struct ProcessConfigurable : Configurable<bool, ConfigParamKind::kProcessFlag> {
   (As...);
 };
 
+template <typename T>
```

Comment:
It might be a good idea to check the existence of the function pointer rather than checking for base_of_template.